src/scss/_sphinx-bootstrap.scss

```diff
@@ -0,0 +1,155 @@
+/*
+ * nature.css_t
+ * ~~~~~~~~~~~~
+ *
+ * Sphinx stylesheet -- nature theme.
+ *
+ * :copyright: Copyright 2007-2011 by the Sphinx team, see AUTHORS.
+ * :license: BSD, see LICENSE for details.
+ *
+ */
+
+// @import url('basic.css');
+
+/* -- page layout ----------------------------------------------------------- */
+
+hr {
+  border: 1px solid #b1b4b6;
+}
+
+/* -- body styles ----------------------------------------------------------- */
+
+body {
+  font-family: sans-serif;
+}
+
+a {
+  color: #005b81;
+  text-decoration: none;
+}
+
+a:hover {
+  color: #e32e00;
+  text-decoration: underline;
+}
+
+/* section header in docstring pages */
+p.rubric {
+  border-bottom: 1px solid rgb(201, 201, 201);
+}
+
+a.headerlink {
+  color: #c60f0f;
+  font-size: 0.8em;
+  padding: 0 4px 0 4px;
+  text-decoration: none;
+}
+
+a.headerlink:hover {
+  background-color: #c60f0f;
+  color: white;
+}
+
+div.deprecated p {
+  display: inline;
+}
+
+div.deprecated {
+  margin-bottom: 10px;
+  margin-top: 10px;
+  padding: 7px;
+  background-color: #ffe4e4;
+  border: 1px solid #f66;
+}
+
+div.highlight {
+  background-color: white;
+}
+
+div.topic {
+  background-color: #eee;
+}
+
+pre {
+  padding: 10px;
+  background-color: rgb(250, 250, 250);
+  color: #222;
+  line-height: 1.2em;
+  border: 1px solid rgb(201, 201, 201);
+  margin: 1.5em 0 1.5em 0;
+  -webkit-box-shadow: 1px 1px 1px #d8d8d8;
+  -moz-box-shadow: 1px 1px 1px #d8d8d8;
+}
+
+.viewcode-back {
+  font-family: Arial, sans-serif;
+}
+
+div.viewcode-block:target {
+  background-color: #f4debf;
+  border-top: 1px solid #ac9;
+  border-bottom: 1px solid #ac9;
+}
+
+/**
+ * Styling for field lists
+ */
+
+/* grey highlighting of 'parameter' and 'returns' field */
+table.field-list {
+  border-collapse: separate;
+  border-spacing: 10px;
+  margin-left: 1px;
+  /* border-left: 5px solid rgb(238, 238, 238) !important; */
+}
+
+table.field-list th.field-name {
```

Comment:
in a future PR I guess some of these could be re-written to leverage the SCSS, yeah?

Reply:
Absolutely. In a different branch / future PR I am working on refactoring the different css files into a structured scss setup.